packages/www/pages/app/stream/[id].tsx

```diff
@@ -305,7 +305,7 @@ const ID = () => {
   const playbackId = (stream || {}).playbackId || "";
   const domain = isStaging() ? "monster" : "com";
   const globalIngestUrl = `rtmp://rtmp.livepeer.${domain}/live`;
-  const globalPlaybackUrl = `https://cdn.livepeer.${domain}/hls/${playbackId}/index.m3u8`;
+  const globalPlaybackUrl = `https://livepeercdn.${domain}/hls/${playbackId}/index.m3u8`;
```

Comment:
Hmm funny, I swear I saw this page using the `getIngest` API somewhere...


Hmm It actually does... don't know what's the difference between the 2
https://github.com/livepeer/livepeer-com/blob/1c9390d3490f6f7473d770d967c672599b3e7d51/packages/www/pages/app/stream/%5Bid%5D.tsx#L281